app/helpers/application_helper.rb

```diff
@@ -1,17 +1,19 @@
 # encoding: utf-8
 module ApplicationHelper
   def faces
-    [ "´ ▽ `",
-    "＾ω＾",
-    "＾∀＾",
-    "(´ε`)",
-    "Σ(ﾟДﾟ)",
-    "( ﾟ ヮﾟ)",
-    "ヽ(´ー｀)ノ",
-    "ノ°ο°)ノ",
-    "o(≧∀≦)o",
-    "(づ｡◕‿‿‿‿◕｡)づ",
-    "♡✰☁☼✿⚈‿‿⚈✿☼☁✰♡"]
+    [
+      '´ ▽ `',
+      '＾ω＾',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.